packages/lisk-transactions/src/0_transfer.js

```diff
@@ -12,13 +12,54 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import cryptography from 'lisk-cryptography';
 import { BYTESIZES, TRANSFER_FEE } from './constants';
 import {
-	getAddressAndPublicKeyFromRecipientData,
 	wrapTransactionCreator,
+	validateAmount,
+	validateAddress,
+	validatePublicKey,
 } from './utils';
 
 const createAsset = data => {
+	if (data && data.length > 0) {
+		return { data };
+	}
+	return {};
+};
+
+const validateInputs = ({ amount, recipientId, recipientPublicKey, data }) => {
+	if (!validateAmount(amount)) {
+		throw new Error(
+			'Please provide an amount. Expected a valid number in string format.',
```

Comment:
We should be consistent with the structure of our messages. How about `Could not create transaction: Amount must be a valid number in string format.`? (Also applies to the other messages.)